src/List.test.jsx

```diff
@@ -0,0 +1,43 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  const handleClickDelete = jest.fn();
+  const renderComponent = ({ tasks }) => render(<List
+    tasks={tasks}
+    onClickDelete={handleClickDelete}
+  />);
+
+  context('with no tasks', () => {
+    let tasks;
+    beforeAll(() => {
+      tasks = [];
+    });
+    it('renders empty message', () => {
+      const { container } = render(<List tasks={tasks} />);
+
+      expect(container).toHaveTextContent('할 일이 없어요!');
+    });
+  });
+
+  context('with tasks', () => {
+    let tasks;
+    beforeAll(() => {
+      tasks = [{ id: 1, title: '일어나기' }];
+    });
```

Comment:
이렇게 사용하려면 `tasks`가 여기 있으면 안됩니다.
```suggestion
describe('List', () => {
  let tasks;
  
  context('with tasks', () => {
    beforeEach(() => {
      tasks = [{ id: 1, title: '일어나기' }];
    });

    it('renders tasks', () => {});    
  });

  context('with no tasks', () => {
    beforeEach(() => {
      tasks = [];
    });
    
    it('renders empty message', () => {});    
  });
```

Reply:
말씀해주신대로 tasks 변수를 context가 아닌 description으로 이동시켰습니다:)